### ISSUE TITLE
Issue - Image reception fails from another users belonging to another server

### ISSUE THREAD

**Issue by FeisTy-H** [ChatSecure/ChatSecure-iOS#1042]
Hi,
Please find below the issue : 

## Context : 
A Conference room is created from another friend on his XMPP server. The room contains users from his server and users from my XMPP server. The room is shared between our respectives users. All messages are sent and received correctly by both parties without any issues. When I send a picture, all participants received and can read the image. When a participant from my server send a picture I can read the image. In contrary when a partipant belonging to the other server send a picture I cannot download it and the preview fails.

## Issue Description : 
On the conversation view I can see the image download progess until 100% but then an error message appears « Request expire ». Even with the error I can’t open the URL on SAFARI.

## iOS device/version : 
Iphone 6S, latest iOS version 11.4.1

## Note :
the issue still persist on 1 to 1 conversation not only the conference room.

*Do you have any idea ? Did you have any feedback about this behavior ?*

**Comment by FeisTy-H** [comment]
I am the only one on iOS and using ChatSecure. All others people uses Conversations and it works fine without any issues.

**Comment by licaon-kter** [comment]
Only you can't receive from the other server? All the others that use Conversations receive it ok?

MUC w/ OMEMO or unencrypted?

**Comment by FeisTy-H** [comment]
Yes only pictures that are sent by users from the remote server failed. 

Conference room is Unencrypted, 1to1 conversation is OMEMO but the result is the same. If it can help please find enclosed the error into the application.

![error_conversation](https://user-images.githubusercontent.com/22450865/43138231-75c889ac-8f4e-11e8-912e-5b2d38c7c7d6.PNG)
![error_detail](https://user-images.githubusercontent.com/22450865/43138236-77e0c2a4-8f4e-11e8-931d-94f2beeb1553.PNG)

**Comment by licaon-kter** [comment]
I saw that error when battery was low and background sync was disabled by Apple, pressing download worked ok though.

**Comment by FeisTy-H** [comment]
not for me. The issue still persist with battery and push activated. when I press download for a new picture the issue persist.

**Comment by FeisTy-H** [comment]
Somebody have an idea ? How can I make more tests to help the troubleshoot ?

**Comment by mimi89999** [comment]
@FeisTy-H Please test this in the latest version.

**Comment by FeisTy-H** [comment]
Hi, which version ? I am not enrolled in beta program but I just installed testflight app if needed.

**Comment by mimi89999** [comment]
@FeisTy-H Last stable version on the AppStore.

**Comment by FeisTy-H** [comment]
Hi, as far I remember the exact issue and the test conducted tonight yes it seems now to be ok. I don't succeed to reproduce the issue.
